### ISSUE TITLE
Investigate loss spikes during training

### ISSUE THREAD

**Issue by 0xtristan** [sholtodouglas/learning_from_play#12]
Hypothesis is that this is to do with potential VAE latent space collapses.
 
Things to try: 1. different activations, 2. lower learning rate, 3. infoVAE, 4. 5d rotations? (else email Corey)

**Comment by 0xtristan** [comment]
Seems like changing the loss function from MSE to MAE (reconstruction loss) makes this wayy better. From what I can tell MSE is quite tricky to optimise for small values, and given that most of our actions are on the interval [-0.2,0.2] this would make sense. Other options are BCE loss if on interval [0,1], but we decided to just stick with MAE because it's simple and seems to work quite well for now.
